includes/Template_Post_Type.php

```diff
@@ -91,6 +91,7 @@ public function register() {
 					'excerpt',
 					'thumbnail', // Used for poster images.
 				],
+				'rewrite'               => false,
```

Comment:
I am guessing these rewrites are not needed?

Reply:
Correct, these are not public post types